docs/install/index.md

```diff
@@ -11,7 +11,7 @@ weight: 1
   * 🐳 [Docker Compose (✨ Recommended)](./installation/docker_compose_install.md)
   * 🦦 [Container (Podman)](./installation/container_install.md)
   * 🐧 [Linux](./installation/linux_install.md) 
-  * 🍎 [Mac](./installation/mac_install.md) 
+  *  [Mac](./installation/mac_install.md) 
```

Comment:
I thought this might be the case, it seems the unicode doesn't render correctly: 
![image](https://github.com/danny-avila/LibreChat/assets/110412045/c6d492b8-91a6-42f9-9c82-d85ecfe496d4)

Reply:
Fixed in the last commit, thanks for your feedback :)